bark_core/signatures/commands/approve_cmd.py

```diff
@@ -69,56 +66,44 @@ def approve(ctx, commit, gpg_key_id, ssh_key_path):
     project = ctx.obj["project"]
     repo = project.repo
 
+    key = None
+
     if not gpg_key_id and not ssh_key_path:
-        key = get_key_from_git(repo)
+        config = repo.config
+        if "user.signingkey" in config:
+            identifier = config["user.signingkey"]
+            type = config["gpg.format"] if "gpg.format" in config else "openpgp"
```

Comment:
Can't this be written as just `type = config.get("gpg.format", "openpgp")`?

Reply:
I tried, but `config` does not expose the `get` method. It is not a `dict`.